ci_framework/roles/edpm_build_images/defaults/main.yml

```diff
@@ -54,3 +54,4 @@ cifmw_edpm_build_images_dry_run: false
 cifmw_edpm_build_images_push_registry: 'quay.rdoproject.org'
 cifmw_edpm_build_images_push_registry_namespace: 'podified-main-centos9'
 cifmw_edpm_build_images_push_container_images: false
+# cifmw_edpm_build_images_cert_install_command: "dnf -y install <path to cert rpm>"
```

Comment:
why this is commented?

Reply:
We no longer need it, we can replace it with
```
cifmw_edpm_build_images_cert_path:  /etc/pki/ca-trust/source/anchors/rh.crt
```